src/main/java/de/codecrafter47/taboverlay/config/placeholder/CustomPlaceholderColorAnimation.java

```diff
@@ -87,6 +89,7 @@ private void updateReplacement() {
             TextColor b = colors.get((ia + 1) % colors.size());
             TextColor c = TextColor.interpolateSine(a, b, sd - ia);
             sb.append(c.getFormatCode());
+            sb.append(sanitizeFormats(formats));
```

Comment:
I'd call the sanitize method in the configuration class, so that is is not called in hot code.

Reply:
Changed